cfgov/unprocessed/js/modules/util/behavior.js

```diff
@@ -30,6 +30,60 @@
 var standardType = require( './standard-type' );
 var dataHook = require( '../../modules/util/data-hook' );
 
+
+/**
+ * @param {string} behaviorSelector
+ *  Behavior type used to find the element within the dom.
+ * @param {HTMLNode} baseElement Containing element for the behavior element.
+ * @returns {HTMLNodeList} behaviorElements if it exists in the dom,
+ *                         null otherwise.
+ */
+function _findElements( behaviorSelector, baseElement ) {
+  baseElement = baseElement || document;
+  var behaviorElements = [];
```

Comment:
Does it need an initial value if `return behaviorElements || [];` happens on L:57?